docs/UserGuide.adoc

```diff
@@ -156,6 +156,11 @@ Views all details of the 1st person in the results of the `find` command.
 Clears all entries from the address book. +
 Format: `clear`
 
+== Sort entries by name: `sort`
+
+Prints out a list of people sorted by name
```

Comment:
Good work on updating the user guide. However, you might wish to add a full stop at the end of the sentence to ensure consistency with the rest of the user guide.